src/lib.rs

```diff
@@ -23,10 +23,16 @@ pub struct World {
 
 impl World {
     pub fn new((width, height): Cell, rule: BigInt) -> World {
+        let rule_reverse = BitVec::from_bytes(&rule.to_bytes_be().1);
+        let mut rule = BitVec::from_elem(512, false);
+        for i in 0..rule_reverse.len() {
```

Comment:
Actually, now that I think of it better, if the problem is you need to reverse the integer sequence, isn't it better to just do `&rule.to_bytes_le()` to translate the BigInt to little-endian format, which is the same as inverting it's big-endian variant?

Reply:
Unfortunately, `to_bytes_le` doesn't do the right thing because the bit order within each byte is still big-endian. So the easiest thing I could think of was to use big-endian _byte_ order, then convert to a `BitVec` to get big-endian _bits_, and then reverse the `BitVec` itself.

Reply:
Maybe consider using [`reverse()`](https://doc.rust-lang.org/std/primitive.slice.html#method.reverse) then?